components/Button/src/index.css

```diff
@@ -96,6 +96,21 @@
   padding-inline-start: var(--denhaag-button-large-size-padding-inline, var(--denhaag-button-padding-inline));
 }
 
+.denhaag-button--outline-dark-action {
+  border-style: solid;
+  background-color: var(--denhaag-button-outline-dark-action-background-color);
+  border-color: var(--denhaag-button-outline-dark-action-border-color);
+  border-width: var(--denhaag-button-border-width);
+  color: var(--denhaag-button-outline-dark-action-color);
+}
```

Comment:
This button style is missing a fix which you can see applied to the secondary action. Due to this variant having a border (like the secondary action), the focus outline will overlap the border.